content/projects/coursera/it-automation/course-2-operating-system/_index.md

```diff
@@ -1,11 +1,10 @@
 ---
-_db_id:
 available_flavours:
 - none
 content_type: project
 prerequisites:
   hard:
-  - projects/coursera/it-automation/course-1-python-crash-course
+  - content/coursera/it-automation/course-1-python-crash-course
```

Comment:
when using content links you need to leave out the `content/` so this should be `coursera/it-automation/course-1-python-crash-course`

Its also important to move the refered to content into the correct folders, this is still in the projects directory and should be int the main content directory. Apologies if my previous review was unclear on that.

Reply:
please let me know when you have made this change, also make sure the change is made on they syllabus

Reply:
Hey,

I already did. Sorry.